app/services/hyrax/workflow/tombstone_approval_notification.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Hyrax
+  module Workflow
+    # Tombstone Approval Notification
+    class TombstoneApprovalNotification < AbstractNotification
+      private
+
+      def subject
+        'Requested Tombstone has been approved'
+      end
+
+      def message
+        "The Tombstone request for #{title} (#{link_to work_id, document_path}) was approved by #{user.user_key}. #{comment}"
+      end
+
+      def users_to_notify
+        user_key = ActiveFedora::Base.find(work_id).depositor
```

Comment:
Im sure it can. This is just copied from the other approval notification we have. https://github.com/OregonDigital/OD2/blob/master/app/services/hyrax/workflow/deletion_approval_notification.rb#L18

If we want this updated, im cool with doing it, I'll update it there as well. Up to you, im fine with either.

Reply:
Yes please 🙂 I missed that one